src/payment-profile/service.js

```diff
@@ -6,7 +6,31 @@ angular
 
       var PaymentService = Service.extend(Payment);
 
+      PaymentService.prototype.addPaymentMethodNonce = function(options) {
+        var data = {
+          paymentMethodNonce : options.paymentMethodNonce,
+        };
+
+        return this.create(data).$promise;
+      };
+
+      PaymentService.prototype.setDefaultPaymentProfile = function(options) {
+
+        var profileId = options.id;
+        
+        if (!profileId) {
+          throw "An existing profile is required. Missing options.id";
+        }
+        var data = {
+          default : true
+        };
+        
+        return this.update(profileId, data).$promise;
+      };
+
       PaymentService.prototype.addPaymentProfile = function(options) {
```

Comment:
You can add a `@deprecated` to a jsdoc here to make it clear that people shouldn't use this when they are reading the docs